src/components/Misc/RsvProgressBar.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  current: string;
+};
+
+const states = {
+  idea: 'Idee',
+  agreement_process: 'Prüfung',
+  planning: 'Entwurf',
+  in_progress: 'Umsetzung',
+  done: 'Abgeschlossen',
+};
+
+const baseStyle =
+  'inline-flex items-center justify-center py-3 px-3.5 lg:px-2.5 text-xs lg:text-xs sm:text-base xl:py-2.5 xl:px-4 font-bold mr-1 xl:text-lg hover:cursor-default';
+const selectedStyle = 'bg-slate-900 text-white';
+const unselectedStyle = 'bg-gray-300 text-gray-600';
+
+export const RsvProgressBar: React.FC<Props> = ({ current }) => {
+  const stateList = Object.keys(states);
+  const nStates = stateList.length;
+  return (
+    <div className="flex flex-row">
+      {stateList.map((state, i) => {
+        const style = classNames(
+          baseStyle,
+          state === current ? selectedStyle : unselectedStyle,
+          i === 0 && 'rounded-l-full',
+          i === nStates - 1 && 'rounded-r-full'
```

Comment:
Ich habe das kürzlich auch so gemacht, aber eigentlich müsste es auch gut mit den TW classes gehen für first-child/last-child:

```suggestion
          'first:rounded-l-full',
          'last:rounded-r-full'
```

Reply:
cool, die kannte ich noch nicht. Habs angepasst